Embedded Metadata.js

```diff
@@ -563,8 +563,9 @@ function addLowQualityMetadata(doc, newItem) {
 	if(newItem.title) {
 		newItem.title = newItem.title.replace(/\s+/g, ' '); //make sure all spaces are \u0020
 		if(newItem.publicationTitle) {
-			//remove publication title from the end of title
-			var removePubTitleRegex = new RegExp('\\s*\\W\\s*'
+			//remove publication title from the end of title (see #604)
+			//this can occur if we have to doc.title, og:title etc.
+			var removePubTitleRegex = new RegExp('\\s*[\-–—=_:|~#]\\s*'
```

Comment:
The `\-` would need a double backslash, but because the `-` is the first character in the group, it does not need to be escaped at all.